content/rc/databases/flush-data.md

```diff
@@ -29,3 +32,35 @@ Example:
 ```sh
 redis-cli -h redis-12345.c24.us-east-mz-1.ec2.cloud.redislabs.com -p 12345 -a xyz flushall
 ```
+
+## Memcached Enterprise cloud subscriptions
+
+If you're using Memcached Enterprise Cloud subscription, you likely cannot use the `nc` command or the `flush_all` command.
```

Comment:
Should "using Memcached Enterprise Cloud subscription" be changed to "using a Memcached Enterprise Cloud subscription"? This is the only potential issue I see. Everything else looks good to me.

Reply:
Yep; fixed.  

(Wa initially confused because I thought you were referring to the header, which looked fine to me.  _Then_ I found the actual typo.)